content/posts/getting-started/github-pages/index.fr.md

```diff
@@ -5,11 +5,11 @@ menu:
   sidebar:
     name: Déployer dans Github Pages
     identifier: getting-started-github
-    parent: getting-started
+    parent: Démarrer
```

Comment:
You can not translate this identifier. This defines the article structure. This probably causing duplicate item on the left sidebar.

Reply:
If I change "Démarrer" to "getting-started", the left menu entry will not be translated into French. exact ?

Reply:
You can only translate the `name` which will be shown on the left sidebar. The parent is use for article structure. It won't be showing anywhere.

Reply:
OK